meetings/2021-03-03.md

```diff
@@ -27,7 +27,7 @@ https://meet.jit.si/solid-authorization
 ### [March 10: 15:00 & 18:00 UTC: Solid/CCG meet & greet with Tim Berners Lee](https://lists.w3.org/Archives/Public/public-credentials/2021Feb/0062.html)
 
 Dmitri: W3C Credentials CG is a large community. It incubated DID WG, VC WG and now it incubates other task forces. Solid community expressed interest in using VC, signatures and encryptions. Next Wednesday they are coming to present during our weekly call and we will present on the same day during theirs.
-...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity pased access control.
+...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity passed access control.
```

Comment:
```suggestion
...: We should decide what we want to present. I'd like us to select a couple of topics. For example, our identity past access control.
```

Reply:
I think that should be identity - "based"